packages/common/src/models/AudioRewards.ts

```diff
@@ -53,6 +53,7 @@ export enum ChallengeName {
   FirstTip = 'ft',
   FirstPlaylist = 'fp',
   ListenStreak = 'l',
+  ListenStreakEndless = 'e',
```

Comment:
is it possible to use `le`for this or are we too far gone?

Reply:
possible but i think minimize num chars = good